scripts/build-catalogue.in

```diff
@@ -108,8 +108,12 @@ with TemporaryDirectory() as tmp:
         fd.write(cmake)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/BuildModules.cmake', tmp)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/generate_catalogue', tmp)
-    sp.run('cmake ..', shell=True, check=True, capture_output=not verbose)
-    sp.run('make',     shell=True, check=True, capture_output=not verbose)
+    if verbose:
+        sp.run('cmake ..', shell=True, check=True, stdout=sp.PIPE, stderr=sp.PIPE)
```

Comment:
This pattern _captures_ the output streams, but it does not do anything with them. 
```py
In [1]: import subprocess as sp

In [2]: sp.run('ls', shell=True, check=True, stdout=sp.PIPE, stderr=sp.PIPE)
Out[2]: CompletedProcess(args='ls', returncode=0, stdout=b'...', stderr=b'...')

In [3]: p = sp.run('ls', shell=True, check=True, stdout=sp.PIPE, stderr=sp.PIPE)

In [4]: p.stdout
Out[4]: b'...'

In [5]: print(p.stdout.decode('utf8'))
...
...
```
Assigning to `p` and decoding/printing the pipes contents is what you actually want to do here, I think.

Reply:
Actually, I needed to swap the if block around: by default things are already printed. Not setting `verbose` should simply hide them.

Reply:
ok, that seems weird to me. I'll take a look.